### PR TITLE
Set SSH timeout to 5 minutes

### PR DESCRIPTION
**Pull request by xueweiz**
This is for https://github.com/kubernetes/node-problem-detector/issues/296#issuecomment-532887983

### CODE REVIEW

File: test/e2e/lib/gce/instance.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ func CreateInstance(instance Instance, imageName string, imageProject string) (I
 	}
 
 	instanceRunning := false
-	for i := 0; i < 30 && !instanceRunning; i++ {
+	for i := 0; i < 15 && !instanceRunning; i++ {
```
Comment (wangzhen127) [inline test/e2e/lib/gce/instance.go]: Can you add a comment before this line on the timeout (15*20=300sec=5min)?

Comment (xueweiz) [reply test/e2e/lib/gce/instance.go]: Sorry. Just changed it. WDYT?

**Pull request merged**